spec/controllers/jasmine_fixtures/contacts_spec.rb

```diff
@@ -20,6 +20,7 @@
     end
 
     it "generates the aspects_manage_contacts_json fixture", fixture: true do
+      alice.disconnect(User.first.contact_for(bob.person))
```

Comment:
`User.first` could fail in the future because `alice` doesn't need to be the `first` user.

But I think it's better to add a third non-mutual user instead of converting one of the existing two contacts a non-mutual. That way we can still check if it still works for multiple mutual users and not just returns one user.

So that would be something like that instead:

```ruby
bob.share_with(FactoryGirl.create(:person), @aspect)
```

Reply:
Done.